app/utils/dictionary/year-list.ts

```diff
@@ -0,0 +1,11 @@
+let year: number = new Date().getFullYear();
+const earliestYear = 1970;
```

Comment:
The issue https://github.com/fossasia/open-event-frontend/issues/8784 lists 1930 as the start year?

Reply:
Hi @norbusan,
I changed the starting year to 1930.
Please help me check.